boards/arm/nucleo_f334r8/doc/nucleof334r8.rst

```diff
@@ -0,0 +1,182 @@
+.. _nucleo_f334r8_board:
+
+ST Nucleo F334R8
+################
+
+Overview
+********
+STM32 Nucleo-64 development board with STM32F334R8 MCU, supports Arduino and ST morpho connectivity.
+
+The STM32 Nucleo board provides an affordable and flexible way for users to try out new concepts
+and build prototypes with the STM32 microcontroller, choosing from the various
+combinations of performance, power consumption and features. 
+
+The Arduino™Uno V3 connectivity support and the ST morpho headers allow to expand easily 
```

Comment:
Sigh. I'm on a (never ending) quest to fix trademark identification. Ideally, you only use (tm) and (r) for trademarks you own (the organization your writing for), and indicate third-party trademarks with an asterisk (*).  Hence we use a (tm) for Zephyr, but (generally) not for any other names. But yes, we've got a mixture.   You're correct, on first reference in a file you'd recognize the trademark name with an asterisk (as you've written in your reply), and then you don't have to recognize it again (you can, but our only requirement is to recognize the first use).